src/sass/components/_footer.scss

```diff
@@ -80,3 +82,28 @@ footer {
     }
   }
 }
+
+@media (max-width: 768px) {
```

Comment:
To be consistent with Bootstrap I suggest to use the built-in mixins; in this case, it will be `media-breakpoint-down`. Example usage:
```
@include media-breakpoint-down(md) {
  footer {
    …
  }
  …
}
```